modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectLayoutResourceImpl.java

```diff
@@ -170,6 +171,13 @@ private com.liferay.object.model.ObjectLayoutBox _toObjectLayoutBox(
 
 		String objectLayoutBoxType = objectLayoutBox.getTypeAsString();
 
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-149014")) &&
+			(objectLayoutBoxType != null)) {
+
+			throw new UnsupportedOperationException(
+				"Categorization layout box is not yet supported");
```

Comment:
Hi @seltong, Brian prefers to throw this exception without a message, so just replace to `throw new UnsupportedOperationException();`